setupJest.ts

```diff
@@ -48,6 +49,6 @@ if (CI) {
   });
 }
 
-// Most of our tests are not aware of the `ngDevMode` Ivy and still rely on the old
-// behavior (as it was in the View Engine).
-((global as unknown) as { ngDevMode: undefined }).ngDevMode = undefined;
+if (typeof ɵglobal.ngDevMode === 'undefined') {
```

Comment:
why do we have to force this value?

Reply:
Our tests are using VE still in the root level, VE doesn’t provide this value.